terra2/assetlist.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "terra2",
+    "assets": [
+      {
+        "description": "The native staking token of Terra.",
+        "denom_units": [
+          {
+            "denom": "uluna",
+            "exponent": 0
+          },
+          {
+            "denom": "luna",
+            "exponent": 6
+          }
+        ],
+        "base": "uluna",
+        "name": "Luna",
+        "display": "luna",
+        "symbol": "LUNA",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/luna.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra/images/luna.svg"
```

Comment:
the svg uploaded looks like the old luna logo. Do we have an svg of the new logo?

Reply:
the SVG is not available on github.
removed the svg link